### ISSUE TITLE
Old Version for Windows

### ISSUE THREAD

**Issue by pexaorj** [OpenShot/openshot-qt#544]
Hello guys,

I tried to find in google the old versions of openshot for windows but I did not find anything.

Anyone was the link to old versions?

Thanks to all, the openshot is amazing!

**Comment by ankitpati** [comment]
Is there any specific need for an older version? On Windows, it is best for users to stick to the latest stable release, which may be downloaded from [here](https://github.com/OpenShot/openshot-qt/releases/download/v2.2.0/OpenShot-v2.2.0-x86_64.exe "OpenShot 2.2").

**Comment by pexaorj** [comment]
Is because when I try to use the latest version, it crash when I open.
So, I just try to use the first version, because that one works fine for me.

**Comment by chuckkh** [comment]
Hmm, no answer for the original question?
I also would like to find old versions, which doesn't seem a strange request for FOSS.
To answer ankitpati's question, my reason is that the latest Debian, Stretch, has Openshot 1.4.3, and I have created a fairly complicated project using it, that I now need to edit in Windows. The file format changed during that time - I get a list of errors and it fails to open - and the interface has also grown more complicated; I need several clicks to simply mute the audio of a clip, e.g. I noted the weaknesses of 1.4.3 but I was able to work around them.
Does any version of 1.4.3 still exist out there, outside of Debian's repositories?
